pymeasure/display/console.py

```diff
@@ -27,45 +27,61 @@
 import os
 import copy
 import argparse
+
 try:
-    import progressbar
-    # Check that progressbar is progressbar2
-    progressbar.streams
+    from tqdm import tqdm
 except (AttributeError, ImportError):
-    progressbar = None
+    tqdm = None
 from .Qt import QtCore
 import signal
 from ..log import console_log
-from .listeners import Monitor
 
-from ..experiment import Results, Procedure, Worker, unique_filename
+from .browser import BaseBrowserItem
+from .manager import BaseManager, Experiment
+
+from ..experiment import Results, Procedure, unique_filename
 
 log = logging.getLogger(__name__)
 log.addHandler(logging.NullHandler())
 
 
+class ConsoleBrowserItem(BaseBrowserItem):
+
+    def __init__(self, progress_bar):
+        self.bar = progress_bar
+
+    def setStatus(self, status):
+        if self.bar:
+            self.bar.set_description(self.status_label[status])
+
+    def setProgress(self, status):
+        if self.bar:
+            self.bar.n = status
+            self.bar.refresh()
+
+
 class ConsoleArgumentParser(argparse.ArgumentParser):
     special_options = {
-        "no-progressbar":   {"default": False,
-                             "desc": "Disable progressbar",
-                             "help_fields": ["default"],
-                             "action": 'store_true'},
-        "log-level":        {"default": 'INFO',
-                             "choices": list(logging._nameToLevel.keys()),
-                             "desc": "Set log level (logging module values)",
-                             "help_fields": ["default"]},
-        "sequence-file":    {"default": None,
-                             "desc": "Sequencer file",
-                             "help_fields": ["default"]},
+        "no-progressbar": {"default": False,
+                           "desc": "Disable progressbar",
+                           "help_fields": ["default"],
+                           "action": 'store_true'},
+        "log-level": {"default": 'INFO',
+                      "choices": list(logging._nameToLevel.keys()),
+                      "desc": "Set log level (logging module values)",
+                      "help_fields": ["default"]},
+        "sequence-file": {"default": None,
+                          "desc": "Sequencer file",
+                          "help_fields": ["default"]},
```

Comment:
That is the default behavior with my ide pycharm when auto-formatting the code, it seems to align the keys. I'll try to find that setting, I agree it can be distracting from a commit.